config/public.pem

```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
+MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAuGhk5EoG0ujwKcqU3bUo
+hUnncDQZgoGTbMt4vXXYRRklFlU7ENPrQ0nqzfWpvJl+SQoJMNMjiYbZ/o200sID
+xBo/+AR5hUFlmcOg9q1/7cJlGQpgv0TnfbSaE85ayUd4IMDcjYjc9YPFBSsGAW4D
+/ofHxUV/N8zIoCyxdBj6cX/bnH8JmkB0alp3P/R6M27i4zn8Q/G3knt4gqUeenpd
+unI7vUtbZg7MCIEwgJrP+QlL25OiEBibTkDc6U2Eg8e6FWloWADID0kcXilLwPtd
+zkYfgkmU/7zIRmSzHXT7eMSpEEDFSbxBSVt4dxKULInrRKWAZI9kqhX2BzZAZeqY
+JQIDAQAB
+-----END PUBLIC KEY-----
```

Comment:
This has to be loaded via an environment variable or at least removed from the version control. Also what exactly is this used for?

Reply:
sure, I mistakenly added this file

Reply:
this was generated by the opensssl command as written in SETUP.md

Reply:
This is for api authentication, no need to put this in public repo, can be gitignored

Reply:
yah @tachyons I will delete it

Reply:
deleted the file